propagators/opencensus_binary/binary.go

```diff
@@ -0,0 +1,86 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
I would prefer to name the directory just `opencensus` and update the package name to match. That way the use of this propagator will be `opencensus.Binary`.

Reply:
done